examples/next-prisma-starter/src/utils/trpc.ts

```diff
@@ -122,6 +122,10 @@ export const trpc = createTRPCNext<AppRouter, SSRContext>({
 
     return {};
   },
+  /**
+   * @link https://trpc.io/docs/v11/data-transformers
+   */
+  transformer,
```

Comment:
this shouldn't be here???

Reply:
It's needed if you use getServerSideProps or getStaticProps to pre-fetch data

We use transformers to serialize/deserialize the cache